index.js

```diff
@@ -327,11 +313,13 @@ module.exports = function svelte(options = {}) {
 					}
 				}
 
-				const writer = new CssWriter(result, {
+				const filename = Object.keys(bundle)[0].split('.').shift() + '.css';
```

Comment:
Should this do it for each element of the array? If not, it might be useful to add a comment explaining why just `[0]`